.github/workflows/ci.yml

```
@@ -424,6 +424,7 @@ jobs:
           echo $GITHUB_RUN_ID | tee src/deployment/.build.id
           echo $GITHUB_SHA | tee src/deployment/.sha
           ./src/ci/get-version.sh > src/deployment/VERSION
+          sed -i "s/0.0.0/$(cat src/deployment/VERSION)/g" src/deployment/bicep-templates/function-settings.bicep
```

Perhaps pattern-match the periods precisely:
```suggestion
          sed -i "s/0\.0\.0/$(cat src/deployment/VERSION)/g" src/deployment/bicep-templates/function-settings.bicep
```